storage/client/blob/pom.xml

```diff
@@ -49,7 +49,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.0.0-preview.2</version>
+      <version>1.0.0-preview.1</version>
```

Comment:
Do we want to go back to preview.1 for this? I thought preview 2 had a fix to address an issue in storage. Please check with @anuchandy